proposals/0370-modifiers.rst

```diff
@@ -54,37 +54,40 @@ Proposed Change Specification
    the Haskell 2010 Report for all BNF syntax, and recall that we use braces
    to denote "zero or more")::
 
-     type     ::= btype [ {modifier} -> type ]
-     prefix%  ::= '%'    -- only in prefix position
-     modifier ::= prefix% atype
+     type          ::= btype [ {modifier} -> type ]
+     prefix%       ::= '%'    -- only in prefix position
+     modifier      ::= prefix% atype
+     modifiers     ::= {modifier}
+     declModifiers ::= [ modifier [ ';' ] [ declModifiers ] ]
```

Comment:
The proposal as-is allows

```
%One
  %Two
data F

%One
  %Two
  data F
```

but forbids

```
%One
%Two
data F

%One
%Two
  data F
```

I propose that these be allowed too. The new `declModifiers` allows semis in between modifiers that precede declarations.